.github/workflows/matchers/pylint.json

```
@@ -0,0 +1,18 @@
+{
+    "problemMatcher": [
```

```suggestion
  "problemMatcher": [
```

Can we use 2 spaces for indent?

---

Can we apply the same fix to other lines?

---

Sorry. I will fix the indent.